makefile

```diff
@@ -38,9 +38,10 @@ run-sim2play: all
 run-release: all-release
 	./run/soccer
 run-sim-release: all-release
-	-pkill -f './simulator --headless'
-	./run/simulator --headless &
-	./run/soccer -sim -pbk example.pbk
+	-pkill -f './grsim'
+	./run/grsim &
+	./run/soccer -sim
```

Comment:
I was intending it to be used during comp (so when we do a release run, it would set up playbooks automatically for us), but I'm fine with this being removed.